src/typescript/frontend/src/components/loading.tsx

```diff
@@ -1,3 +1,5 @@
+"use client";
+
```

Comment:
How come you added this here? I think it's a client component by necessity and thus by default. Just curious

Reply:
It was such a while ago I can't exactly remember. Probably I got an error without it.